example/spec/parallel/syncGranule/SyncGranuleDuplicateHandlingSpec.js

```diff
@@ -278,18 +277,18 @@ describe('When the Sync Granule workflow is configured', () => {
       it('captures both files', async () => {
         // This assertion is to check that the granule has been updated in dynamo
         // before performing further checks
-        const record = await waitForModelStatus(
-          granuleModel,
-          { granuleId: inputPayload.granules[0].granuleId },
-          'completed'
+        const granule = await waitForApiRecord(
+          getGranule,
+          {
+            prefix: config.stackName,
+            granuleId: inputPayload.granules[0].granuleId,
+          },
+          {
+            status: 'completed',
```

Comment:
So the fix in this PR was due to a situation where the found granule record was completed from a different execution?   Nice catch -- can we get a quick justification for the PR in the PR description or a ticket for posterity?

Reply:
Correct. There is no ticket 😅 , but sure I'll update the PR description